instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterBuilder.java

```diff
@@ -64,6 +65,7 @@ public final class InstrumenterBuilder<REQUEST, RESPONSE> {
   final List<AttributesExtractor<? super REQUEST, ? super RESPONSE>>
       operationListenerAttributesExtractors = new ArrayList<>();
   final List<ContextCustomizer<? super REQUEST>> contextCustomizers = new ArrayList<>();
+  final List<InternalShouldStartFilter<? super REQUEST>> shouldStartFilters = new ArrayList<>();
```

Comment:
Is it set as a global filter for all instrumentations or an InstrumenterBuilder-level filter?